test/control-plane/alb-control-plane-stack.test.ts

```diff
@@ -22,6 +22,9 @@ import { SOLUTION_CONFIG_PATH } from '../../src/control-plane/private/solution-c
 import { TestApp, removeFolder } from '../common/jest';
 import { validateSubnetsRule } from '../rules';
 
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+jest.mock('aws-cdk-lib/aws-lambda-nodejs', () => require('../cdk-lambda-nodejs-mock'));
```

Comment:
In the build history of Action, the `build` workflow still cost the same time as previous.

Does it work in ci environment?

Reply:
Use **CI** to determine if it is in the testing environment.